packages/auto-label/src/auto-label.ts

```diff
@@ -18,19 +18,28 @@ import {Storage} from '@google-cloud/storage';
 import {Application} from 'probot';
 // eslint-disable-next-line node/no-extraneous-import
 import {GitHubAPI} from 'probot/lib/github';
-// eslint-disable-next-line node/no-extraneous-import
-import Webhooks from '@octokit/webhooks';
 
 // eslint-disable-next-line @typescript-eslint/no-var-requires
 const colorsData = require('./colors.json');
+// eslint-disable-next-line @typescript-eslint/no-var-requires
```

Comment:
For both `fs` and `path`, you should use `import * as fs from 'fs'` style syntax.